src/Core/Authorization/AuthorizationResolver.cs

```diff
@@ -232,8 +232,7 @@ public void SetEntityPermissionMap(RuntimeConfig runtimeConfig)
                 bool isStoredProcedureEntity = entity.Source.Type is EntitySourceType.StoredProcedure;
                 if (isStoredProcedureEntity)
                 {
-                    SupportedHttpVerb[] methods = entity.Rest.Methods;
-
+                    SupportedHttpVerb[] methods = (entity.Rest.Methods is not null) ? entity.Rest.Methods : Array.Empty<SupportedHttpVerb>();
```

Comment:
Same - shouldn't this methods array be populated with POST?

Reply:
updated